tests/unit/collapsible/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+pret<!doctype html>
```

Comment:
ha ha who knows i must have been trying to type in a different window and did this by accident :-) will remove